test/integration/targets/callback_default/callback_default.out.hide_skipped_ok.stdout

```diff
@@ -31,5 +31,5 @@ TASK [Second free task] ********************************************************
 changed: [testhost]
 
 PLAY RECAP *********************************************************************
-testhost                   : ok=10   changed=7    unreachable=0    failed=0    skipped=1   
+testhost                   : ok=10   changed=7    unreachable=0    failed=0    rescued=0    ignored=1   
```

Comment:
we might want to change to `failed=Y (ignored=X)` so the 'sum of tasks' seems to stay the same, these are not at the same level as the general counters, they are subsets of the existing categories